examples/online_srv/update_online_pred.py

```diff
@@ -71,13 +71,17 @@ def update_online_pred(experiment_name="online_srv"):
 
     online_manager.update_online_pred()
 
+def main(provider_uri = "~/.qlib/qlib_data/cn_data", region=REG_CN, experiment_name="online_srv"):
+    provider_uri = "~/.qlib/qlib_data/cn_data"  # target_dir
+    qlib.init(provider_uri=provider_uri, region=region)
+    first_train(experiment_name)
+    update_online_pred(experiment_name)
 
 if __name__ == "__main__":
     ## to train a model and set it to online model, use the command below

```

Comment:
It will be better if we have a `RollingOnlineExample.run_all`  function for this example.